code/game/machinery/constructable_frame.dm

```diff
@@ -156,6 +156,8 @@
 							else
 								circuit.loc = null
 							new_machine.RefreshParts()
+							if(istype(new_machine, /obj/machinery/conveyor)) //unfortunately I can't do this on the conveyor belt's New() or something actually OOP because I need the user
```

Comment:
There are so much ways to approach this, for example at the circuit board level.

Don't do this.